### PR TITLE
Create 2018-10-31-coreweekly-week-43-2018.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+![Core Weekly banner](/assets/images/2017/04/core_weekly_banner.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, part of the PrestaShop team's developers attended the PHP Forum in Paris. It is a good event to discover what's new with the technology and learn new skills. However, it has an impact on the productivity: less pull requests have been merged than usual.
+
+The recent upgrade of PrestaShop's APIs infrastucrures improved the overall performance of stores back office. In parallel, the Gamification module has been improved to. A new release is available, for both 1.6 and 1.7 versions of PrestaShop:
```
Comment (jolelievre) [inline news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: improved **too**
the release for 1.6 has been published? or is it pending? I am not sure any more

Comment (matks) [inline news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: `infrastucrures` => `infrastuctrures`

Comment (matks) [reply news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: `has been improved to` => `has been improved too` ?

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+* All static files are included in the module, in order to remove a lot of requests to the servers
+* The json data requested by the module is 90% lighter in order to decrease the bandwidth
+* Some users reported an issue with very old versions of PHP; a fix is currently being tested and a another release is expected soon, but please keep in mind that for your shop security and performance, you should use a supported version of PHP
```
Comment (matks) [inline news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: `a another` => `another`

**Review by jolelievre — DISMISSED** [review]

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 22nd to Sunday 28th of October 2018.
+
+![Core Weekly banner](/assets/images/2017/04/core_weekly_banner.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, part of the PrestaShop team's developers attended the PHP Forum in Paris. It is a good event to discover what's new with the technology and learn new skills. However, it has an impact on the productivity: less pull requests have been merged than usual.
```
Comment (LouiseBonnard) [inline news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: However, it has had an impact on productivity: fewer pull requests have been merged than usual.

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md (hunk 1 above)
Comment (Quetzacoalt91) [reply news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: infrastructure is mispelled

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+The recent upgrade of PrestaShop's APIs infrastucrures improved the overall performance of stores back office. In parallel, the Gamification module has been improved to. A new release is available, for both 1.6 and 1.7 versions of PrestaShop:
+
+* All static files are included in the module, in order to remove a lot of requests to the servers
+* The json data requested by the module is 90% lighter in order to decrease the bandwidth
```
Comment (Quetzacoalt91) [inline news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: As far as i know, the JSON is the same but its content is compressed in the answer.

File: news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: ok

Comment (ttoine) [reply news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md]: done. can you please unlock the merge?

**Pull request merged**